segmentation_models_pytorch/encoders/timm_gernet.py

```diff
@@ -0,0 +1,120 @@
+from timm.models import ByobCfg, BlocksCfg, ByobNet
+
+from ._base import EncoderMixin
+import torch.nn as nn
+
+
+class GERNetEncoder(ByobNet, EncoderMixin):
+    def __init__(self, out_channels, depth=6, **kwargs):
+        super().__init__(**kwargs)
+        self._depth = depth
+        self._out_channels = out_channels
+        self._in_channels = 3
+
+        del self.head
+
+    def get_stages(self):
+        return [
+            nn.Identity(),
+            self.stem,
+            *self.stages[:-1],
+            nn.Sequential(self.stages[-1], self.final_conv)
+        ]
```

Comment:
Byobnet has its own stages, forward source here https://github.com/rwightman/pytorch-image-models/blob/d8e69206be253892b2956341fea09fdebfaae4e3/timm/models/byobnet.py#L633-L642

Reply:
please, use explicit stage definition instead of `*` expression